<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calligra::Sheets::TableShapeConfigWidget</class>
 <widget class="QWidget" name="Calligra::Sheets::TableShapeConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Spreadsheet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="createSpreadsheet">
     <property name="toolTip">
      <string comment="@info:tooltip">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Creates an empty spreadsheet that will be saved insaide your main document.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Create empty spreadsheet</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="importSpreadsheet">
     <property name="toolTip">
      <string comment="@info:tooltip">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The imported spreadsheet will be saved inside your main document.&lt;/p&gt;&lt;p&gt;Note that changes will not be saved to the external document.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Import spreadsheet from file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="linkSpreadsheet">
     <property name="toolTip">
      <string comment="@info:tooltip">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Creates a link to an external spreadsheet.&lt;/p&gt;&lt;p&gt;Note that the spreadsheet is editable, but changes will not be saved to the external file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Link to external spreadsheet file</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="url"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
